components/backline/plan.sh

```diff
@@ -0,0 +1,41 @@
+pkg_name=hab-backline
+pkg_origin=core
+pkg_version=$(cat ../../VERSION)
```

Comment:
@reset This differs slightly than the other uses of this - `pkg_version=$(cat "$PLAN_CONTEXT/../../../VERSION")`

I'm not sure I know what the different of a backline is vs other components, but when there's a pattern used in most places and a difference seen, I usually leave a comment as to why it's different.